packages/mdc-chips/README.md

```diff
@@ -257,8 +257,8 @@ Mixin | Description
 `mdc-chip-trailing-icon-color($color, $opacity, $hover-opacity, $focus-opacity)` | Customizes the color of a trailing icon in a chip, optionally customizes regular/hover/focus opacities
 `mdc-chip-leading-icon-size($size)` | Customizes the size of a leading icon in a chip
 `mdc-chip-trailing-icon-size($size)` | Customizes the size of a trailing icon in a chip
-`mdc-chip-leading-icon-margin($top, $right, $bottom, $left)` | Customizes the margin of a leading icon in a chip
-`mdc-chip-trailing-icon-margin($top, $right, $bottom, $left)` | Customizes the margin of a trailing icon in a chip
+`mdc-chip-leading-icon-margin($right, $left)` | Customizes the margin of a leading icon in a chip
```

Comment:
Do you think the parameters needs to be reversed for discoverability? 

Also, the parameters needs to be renamed with valid suffix. For example, `$left-margin`, `$right-margin`.

Reply:
stylelint wants the relevant properties to be used in right, left order (padding-right before padding-left, for example). I find left, right to be more legible though.

Reply:
Param documentation fixed.

Reply:
AFAIK we do not have any lint rules related to order of sass rules.

Maybe stylelint is pointing to the order of padding styles inside that mixin?

```sass
padding-right: $horizontal-padding-value;
padding-left: $horizontal-padding-value;
```

Reply:
That's correct